### PR TITLE
[ZT] WARP client certificate check

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 1]
```diff
@@ -8,16 +8,32 @@ weight: 3
...
 
+{{<details header="Feature availability">}}
+
+| [WARP modes](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes/) | [Zero Trust plans](https://www.cloudflare.com/teams-pricing/) |
+| -- | -- |
+| All modes | All plans  |
+
+| System   | Availability | Minimum WARP version |
+| ---------| -------------| ---------------------|
+| Windows  | ✅           | ?         |
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]: Need WARP version numbers

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 2]
```diff
@@ -58,18 +77,19 @@ $ /usr/bin/security find-certificate -c "<COMMON_NAME>" -p /Library/Keychains/Sy
 {{<tab label="windows" no-code="true">}}
 
 1. Open a PowerShell window.
-2. Run the following command to search for a certificate with a specific common name:
+2. To search the local machine trust store for a certificate with a specific common name, run the following command:
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md:77]: Is this `\LocalMachine\Root`?

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 3]
```diff
@@ -33,8 +49,11 @@ The Client Certificate device posture attribute checks if the device has a valid
 
    1. **Name**: Enter a unique name for this device posture check.
    2. **Operating system**: Select your operating system.
-   3. **Certificate ID**: Enter the UUID of the root CA.
-   4. **Common name**: Enter the common name of the client certificate (not the root CA).
+   3. **OS locations**: On Windows and Linux, select the location(s) where the client certificate is installed.
+   4. **Certificate ID**: Enter the UUID of the root CA.
+   5. **Common name**: (Optional) To check for a specific common name on the client certificate, enter a string with optional `${serial_number}` and `${hostname}` variables (for example, `${serial_number}_mycompany`). WARP will search for an exact, case-sensitive match. If you do not specify a common name, WARP will ignore the common name field on the certificate.
```
Comment (aw-cf) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]:
```suggestion
   5. **Common name**: (Optional) To check for a specific common name on the client certificate, enter a string with optional `${serial_number}` and `${hostname}` variables (for example, `${serial_number}_mycompany`). WARP will search for an exact, case-insensitive match. If you do not specify a common name, WARP will ignore the common name field on the certificate.
```

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 4]
```diff
@@ -33,8 +49,11 @@ The Client Certificate device posture attribute checks if the device has a valid
...
+   6. **Check for extended key use**: (Optional) Check whether the client certificate is scoped to the **Client authentication** and/or **Email** use cases.
```
Comment (aw-cf) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]:
```suggestion
   6. **Check for Extended Key Usage**: (Optional) Check whether the client certificate needs to have one or more attributes set. Supported values are **Client authentication** (`1.3.6.1.5.5.7.3.2`) and/or **Email** (`1.3.6.1.5.5.7.3.4`).
```

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md (hunk 2 above)
Comment (aw-cf) [reply content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md:77]:
```powershell
  PS C:\Users\JohnDoe> Get-ChildItem Cert:\LocalMachine\My\ | where{$_.Subject -like "*<COMMON_NAME>*"}
  ```

File: content/cloudflare-one/identity/devices/warp-client-checks/_index.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ These device posture checks are performed by the [Cloudflare WARP client](/cloud
 | ---------------------| ----- | ------- | ----- | --- | ---------------- |
 | [Application check](/cloudflare-one/identity/devices/warp-client-checks/application-check/) | ✅ | ✅ | ✅ | ❌ | ❌ |
 | [Carbon Black](/cloudflare-one/identity/devices/warp-client-checks/carbon-black/) | ✅ | ✅ | ✅ | ❌ | ❌ |
-| [Client certificate](/cloudflare-one/identity/devices/warp-client-checks/client-certificate/) | ✅ | ✅ | ✅ | ❌ | ❌ |
+| [Client certificate](/cloudflare-one/identity/devices/warp-client-checks/client-certificate/) | ✅ | ✅ | ❌ | ❌ | ❌ |
```
Comment (aw-cf) [inline content/cloudflare-one/identity/devices/warp-client-checks/_index.md]: Client certificate checks are supported on Linux. We should undo this change

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 6]
```diff
@@ -8,16 +8,32 @@ weight: 3
 
 The Client Certificate device posture attribute checks if the device has a valid certificate signed by a trusted certificate authority (CA). The posture check can be used in Gateway and Access policies to ensure that the user is connecting from a managed device.
 
+{{<details header="Feature availability">}}
+
+| [WARP modes](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes/) | [Zero Trust plans](https://www.cloudflare.com/teams-pricing/) |
+| -- | -- |
+| All modes | All plans  |
+
+| System   | Availability | Minimum WARP version |
```
Comment (aw-cf) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]: We should reconsider this table given availability is not a simple ✅ or ❌. Specifics aspects of a check may require a newer WARP version than others

**Review by kodster28 — APPROVED** [review]

File: content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md [hunk 7]
```diff
@@ -8,16 +8,33 @@ weight: 3
...
+
+| [WARP modes](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes/) | [Zero Trust plans](https://www.cloudflare.com/teams-pricing/) |
+| -- | -- |
+| All modes | All plans  |
+
+| System   | Availability | Minimum WARP version<sup>1</sup> |
+| ---------| -------------| ---------------------|
+| Windows  | ✅           | 2024.6.415.0 |
+| macOS    | ✅           | 2024.6.416.0 |
+| Linux    | Coming soon  |   |
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]:
```suggestion
| Linux    | Coming soon  |    |
```

Comment (ranbel) [reply content/cloudflare-one/identity/devices/warp-client-checks/client-certificate.md]: retrigger build
